js/controller.js

```diff
@@ -0,0 +1,5 @@
+(function controller (view){
```

Comment:
You are rendering the views with Node and here this isn't necessary because it's a training of layout (only HTML and CSS without JS)